src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1

```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "cognitiveservices/data-plane/Personalizer" -AutoRestVersion "latest" -SdkGenerationDirectory "$PSScriptRoot\Generated\Personalizer" -ConfigFileTag "release_1_0_preview"
```

Comment:
Is the tag `release_1_0_preview` set to default in readme.md, if yes, this is not needed

Reply:
removed